product_docs/docs/pgd/5/cli/command_ref/pgd_switchover.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: switchover
+---
+
+Switchover to new write leader.
+
+### Synopsis
+
+Switchover to new write leader. Use switchover method fast for immediate
+switchover, or strict to wait until lag is less than route_writer_max_lag
```

Comment:
Make this two sentences:

switchover. Use `strict` to wait until lag is less than `route_writer_max_lag`

Reply:
fixed but with single quote